### PR TITLE
integrating travis

### PR DESCRIPTION
**Pull request by Anies12**
relates #33

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ - npm i
+ - psql -c "create database travis_ci_test;"
+ - psql -c "create user travis with password '123456';" -U
+
+notifications:
+  email: false
+
+env:
+  global:
+  - PGPORT=5432
```
Comment (kannanhassouna) [inline .travis.yml:18]: white space

**Comment by kannanhassouna** [comment]
did you insert db configuration in travis itself

File: .travis.yml (hunk 1 above)
Comment (Anies12) [reply .travis.yml:18]: removed it

**Comment by Anies12** [comment]
I did it 
a minute ago :)

**Review by kannanhassouna — APPROVED** [review]
did you add these configurations to Travis itself?
for merge but you will wait your team review and merge for file structure branch

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - postgresql
+before_script:
+ - npm i
+ - psql -c "create database travis_ci_test;"
+ - psql -c "create user travis with password '123456';" -U
+notifications:
+  email: false
+env:
+  global:
+  - PGPORT=5432
```
Comment (Fatmasiam) [inline .travis.yml:18]: adding new line

**Review by Fatmasiam — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (Anies12) [reply .travis.yml:18]: what do you mean?

**Review by ahmedalmi95 — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Pull request merged**